geometry_msgs/Pose2D position
geometry_msgs/Pose2D relativePosition
string data
float32 yaw
float32 pitch
float32 roll
float32 size
